spm_funcs.py

```diff
@@ -1,65 +1,51 @@
-"""
-This will be your new module defining SPM-like functions
-
-Here you want the get_spm_globals function from the earlier
-``four_dimensions_exercise``, with anything that function imports and other
-definitions that the function needs.
-
-When you are done, you should be able to run this spm_funcs.py module as a
-script and see the script print a message beginning with "OK".
-
-You run this module as a script like this::
-
-    python3 spm_funcs.py
-
-or better, in IPython::
-
-    %run spm_funcs.py
-"""
-
 import numpy as np
-
 import nibabel as nib
-
 import nipraxis
 
+from numpy.typing import ArrayLike
 
-def spm_global(vol):
-    """ Calculate SPM global metric for array `vol`
+
+def spm_global(vol: ArrayLike) -> ArrayLike:
+    """Calculate SPM global metric for array `vol`.
 
     Parameters
     ----------
-    vol : array
+    vol : ArrayLike
         Array giving image data, usually 3D.
 
     Returns
     -------
-    g : float
+    ArrayLike
```

Comment:
Yes, although that is a little confusing, because it is a scalar array (single value).